src/auth/auth.controller.ts

```diff
@@ -57,4 +82,26 @@ export class AuthController {
   ): Promise<User> {
     return this.authService.updateUser(user, updateUserDetailsDto);
   }
+
+  @Post('/upload')
```

Comment:
I think the route should be similar in name to the `@Get('/profile-image/:imageName')`
Create profile image route: `@POST('/upload/profile-image')`